prow/cmd/config-bootstrapper/main.go

```diff
@@ -135,7 +135,7 @@ func main() {
 			logrus.WithError(err).Errorf("Failed to find configMap client")
 			continue
 		}
-		if err := updateconfig.Update(&updateconfig.OSFileGetter{Root: o.sourcePath}, configMapClient, cm.Name, cm.Namespace, data, nil, logger); err != nil {
+		if err := updateconfig.Update(&updateconfig.OSFileGetter{Root: o.sourcePath}, configMapClient, cm.Name, cm.Namespace, data, true, nil, logger); err != nil {
```

Comment:
Can you use constants here to increase readability, rather than true?

Reply:
Done.